forms/AddressLookup/index.js

```diff
@@ -126,6 +126,12 @@ export default React.createClass({
     })
   },
 
+  handleCountrySelectBlur() {
+    this.setState({
```

Comment:
Hrm, added a spec... I need a way to mock components which use the aggregate focus component for specs. I don't want to know about the delay in tests. But will have to have a think about it.